docs/docs/features/mobile-app.mdx

```diff
@@ -27,3 +27,39 @@ The beta release channel allows users to test upcoming changes before they are o
 :::info
 You can enable automatic backup on supported devices. For more information see [Automatic Backup](/docs/features/automatic-backup.md).
 :::
+
+## Album Sync
+
+You can sync or mirror an album from your phone to the Immich server on your account. For example, if you select Recents, Camera and Videos album for backup, the corresponding album with the same name will be created on the server. Once the assets from those albums are uploaded, they will be put into the target albums automatically.
+
+### Album Synchronization Highlights
+
+- **One-Way Sync:** Synchronization is one-way, from the device to the server.
+
+- **Name Matching:** If an album on the server has the same name as the album on the device, images from the device will be merged with the existing images in the server album.
+
+- **Shared Albums:** If the matching album on the server is shared, the new photos merged into the album will also be shared.
+
+- **Album Structure:** When an album is created for the first time, its structure is based on the initial state. Future updates made on the phone (such as deleting or repositioning photos) will not be reflected in Immich.
+
+- **User-Specific Sync:** Album synchronization is unique to each server user and does not sync between different users or partners.
+
+- **Mobile-Only Feature:** Album synchronization is currently only available on mobile. For similar options on a computer, refer to [Libraries](/docs/features/libraries) for further details.
+
+### Synchronizing albums from the past
+
+Albums can be synchronized to the server even if they did not exist on the server before, in order to apply this setting you have to:  
+Enter the cloud on the top right -> cog wheel on the top right -> and select the sync option under Sync albums.
+
+:::info Sync albums delete/move photos
+If you delete/move photos in the local album on your device, it will not be reflected in the album on the server **even if** you click Sync albums
+It will only reflect files you add.
+:::
+
+If the same asset is in more than one album it will only sync to the first album it's in, after that it won't sync again even if the user clicks sync albums manually.
```

Comment:
> first album it's in


I actually don't really know if this will necessarily be the first album he appears on
And if so, how does he know what the first album he appeared on was.